certificate-service/integration-test/src/test/java/se/inera/intyg/certificateservice/integrationtest/fk7211/FK7211ActiveIT.java

```diff
@@ -513,4 +519,216 @@ void shallReturnCertificateIfPatientIsProtectedPersonAndUserIsDoctor() {
       );
     }
   }
+
+  @Nested
+  @DisplayName("FK7211 - Uppdatera svarsalternativ")
+  class UpdateCertificate {
+
+    @Test
+    @DisplayName("FK7211 - Läkare - Om intyget är utfärdat på en patient som har skyddade personuppgifter skall svarsalternativ uppdateras")
+    void shallUpdateDataIfUserIsDoctorAndPatientIsProtectedPerson() {
+      final var testCertificate = testabilityApi.addCertificate(
+          customTestabilityCertificateRequest(FK7211, VERSION)
+              .patient(ANONYMA_REACT_ATTILA_DTO)
+              .build()
+      );
+      final var questionId = "FRG_1";
+      final var expectedDate = LocalDate.now().plusDays(5);
+      final var certificate = Objects.requireNonNull(testCertificate.getBody()).getCertificate();
+
+      Objects.requireNonNull(
+          certificate.getData().put(
+              questionId,
+              updateDateValue(certificate, questionId, expectedDate))
+      );
+
+      final var response = api.updateCertificate(
+          customUpdateCertificateRequest()
+              .user(AJLA_DOCTOR_DTO)
+              .certificate(certificate)
+              .build(),
+          certificateId(testCertificate.getBody())
+      );
+
+      assertEquals(expectedDate,
+          getValueFromData(response.getBody(), CertificateDataValueType.DATE, questionId)
+      );
+    }
+
+    @Test
+    @DisplayName("FK7211 - Vårdadmin - Om intyget är utfärdat på en patient som har skyddade personuppgifter skall felkod 403 (FORBIDDEN) returneras")
+    void shallNotUpdateDataIfUserIsCareAdminAndPatientIsProtectedPerson() {
```

Comment:
We don't need to do any actual updates... just call the api to update and it will give the error.